core/src/main/java/org/mskcc/cbio/portal/scripts/ImportGenePanel.java

```diff
@@ -99,14 +99,17 @@ public void importData() throws Exception {
         String description = getPropertyValue("description", properties, false);
         Set<Integer> genes = getGenes("gene_list", properties, genePanelRepository);        
         
-        GenePanel genePanel = genePanelRepository.getGenePanelByStableId(stableId).get(0);
+		GenePanel genePanel = new GenePanel();
+        List<GenePanel> genePanelResult = genePanelRepository.getGenePanelByStableId(stableId);
         boolean panelUsed = false;
-        if (genePanel != null) {
+        if (genePanelResult != null && genePanelResult.size() > 0) {
```

Comment:
now code checks repository result for empty set before trying to access member 0